x/pool-incentives/spec/01_concepts.md

```diff
@@ -0,0 +1,9 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+The purpose of the `pool incentives` module is to distribute incentives to the pool LPs within the `Gamm` module.
+
+`Pool incentives` module doesn't directly distribute the rewards to the LPs. When a pool is created, the `pool incentives` module creates a `pot` in the `incentives` module for every lock duration that exists. Also, the `pool incentives` module takes a part of the minted inflation from the mint module, and automatically distributes it to the various selected pots.
```

Comment:
Can we rename the module to something like OsmoRewards to specify that this is for base asset rewards specifically

Reply:
Maybe BaseRewards then?

Reply:
I'd like to change this eventually, but right now, not worth the time.